tasks/install_os.yml

```diff
@@ -113,3 +108,23 @@
   set_fact:
     async_install: "{{ async_install|default([]) + [hyp_reachable.ansible_job_id] }}"
 
+- name: Wait for undercloud to be available
+  async_status:
+    jid: "{{ item }}"
+  register: install_uc
+  until: install_uc.finished
+  retries: 300
+  delay: 10
+  with_items: "{{ async_install }}"
+
+- name: upgrade os and reboot
+  block:
+    - name: upgrade os
+      shell: |
+        sshpass -p {{ hypervisor_password }}  ssh -o 'PreferredAuthentications=password' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -q root@{{ hypervisor_host }} '/root/update-latest-rhel-release.sh {{ upgrade_os }} && dnf update -y && reboot'
+      ignore_errors: true
+
+    - include_tasks: tasks/wait_after_node_reboot.yml
+      vars:
+        hypervisor_host: "{{ hypervisor_host }}"
```

Comment:
Why is `hypervisor_host` defined again here when it is also defined in https://github.com/redhat-performance/jetpack/blob/cf41462f75aee5f2ec8199d3838c4f8881be9675/prepare_undercloud.yml#L41? Shouldn't the value of `hypervisor_host` be accessible in the `tasks/wait_after_node_reboot.yml `playbook without this line also?